cli/README.md

```diff
@@ -0,0 +1,20 @@
+<div align="center">
```

Comment:
I would use the same styling as the top level README here:

```
<p align="center">
  <a href="https://grafbase.com">
    <img src="https://grafbase.com/images/other/grafbase-logo-circle.png" height="96">
    <h3 align="center">Grafbase CLI</h3>
  </a>
</p>
```

Reply:
Done